assets/js/index.js

```diff
@@ -7,6 +8,7 @@ require('bootstrap');
 require('select2');
```

Comment:
You could rewrite this as `import 'select2';` without the `from` part to be consistent with the rest of the imports.